### ISSUE TITLE
Android Emulator not responding in windows 11

### ISSUE THREAD

**Issue by mohammedyunusulla** [xamarin/get-started#118]
Problem encountered on https://dotnet.microsoft.com/learn/xamarin/hello-world-tutorial/devicesetup
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

**Comment by mohammedyunusulla** [comment]
![image](https://user-images.githubusercontent.com/31335706/141734563-3f074604-246a-4b73-88ef-89f5eab4aa02.png)

**Comment by mohammedyunusulla** [comment]
![image](https://user-images.githubusercontent.com/31335706/141734693-fece3c14-8a0b-4c9f-834d-adfa32e6ae5d.png)

**Comment by jamesmontemagno** [comment]
Yes, I believe there is a known issue on Windows 11 for some CPU effecting all android emulators with Hyper-V. Would need to use a device, use Intel HAXM, or if you can use WSA -> https://montemagno.com/goodbye-android-emulators-windows-subsytem-for-android-is-here/

**Comment by mohammedyunusulla** [comment]
looks like the Windows subsystem for android is only available for windows
insider and only on US.
any other workaround I can look into?

Regards,
Yunus.

**Comment by jamesmontemagno** [comment]
I talked with the team and this seems to be a global bug across some processors. The teams are looking into it. so not anything specific to VS or Xamarin
